README.md

```diff
@@ -9,18 +9,33 @@ code:
 
 # installation
 
+## using docker
+
+### building (and updating) the docker image
+
+Build the docker image by running `docker build -t knurling-zappa .` in the projects root directory. This only required once or when the zappa version is updated. The resulting container will have awscli, zappa and a basic set of dependencies pre-installed.
+
+### running the docker container
+
+!NOTE! Windows users will need to provide absolute path values in the folling command, because docker. If you're running the command from WSL2, it will work without modification.
+
+Run `docker run -v ```pwd```/:/home/code -v ~/.aws:/root/.aws -eUSERNAME=changeme -eAPI_KEY=changeme -eSECRET_TOKEN=bar -p127.0.0.1:4567:4567/tcp -it knurling-zappa /bin/bash`. Replace github username, API key and secret_token with suitable values.
+
+This will mount the current directory in the path `/home/code` as well as the AWS config directory required by zappa and give you a shell inside the container. It will expose port `4567` used by the application server to the docker host.
+
+You can start the SSO login process outside the container and run `zappa deploy|update|....` inside the container.
+
+## using virtualenv
+
 ## dependencies
 
+Make sure you're operating in a [virtual environment](https://docs.python.org/3/library/venv.html) **with python 3.7**
+
 ```console
 $ pip3 install -r requirements.txt
 ```
 This also installs [Zappa](https://github.com/Miserlou/Zappa) and all the dependencies needed for a Zappa deploy
 
-# deploying
```

Comment:
the `# deploying` section needs to stay (unfortunately I can't create a mergeable suggestion for that)